src/ocm/components/clusterConfiguration/DiscoveryImageForm.tsx

```diff
@@ -26,7 +25,7 @@ const DiscoveryImageForm: React.FC<DiscoveryImageFormProps> = ({
   onCancel,
   onSuccess,
 }) => {
-  const { infraEnv, error: infraEnvError, isLoading } = useInfraEnv(cluster.id);
+  const { infraEnv, error: infraEnvError } = useInfraEnv(cluster.id);
```

Comment:
Why is the loading state not relevant anymore?

Reply:
No, it is not needed because I changed all the loading (part of: https://issues.redhat.com/browse/MGMT-9961)

Reply:
Makes sense, thanks